src/app/entities/auth/components/login-form/login-form.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AppRoutePath } from '@app/enums/app-route-path.enum';
+import { AuthService } from '@app/services/auth/auth.service';
+
+@Component({
+  selector: 'app-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: [ './login-form.component.scss' ],
+})
+export class LoginFormComponent {
+  email: string;
+  password: string;
+
+  constructor(private router: Router, private authService: AuthService) { }
+
+  onFormSubmit() {
+    this.login();
+  }
+
+  private login() {
+    this.authService.login$(this.email, this.password).subscribe(
```

Comment:
same here..
+ it is goot to pass obect as an input param: `this.authService.login({ email: this.email, password: this.password });`

Reply:
Agree